SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -121,6 +141,7 @@ struct FConnectionConfig
 	uint32 WorkerSDKLogFileSize;
 	Worker_LogLevel WorkerSDKLogLevel;
 	Worker_NetworkConnectionType LinkProtocol;
+	TMap<EWorkerType, Worker_NetworkConnectionType> ConnectionTypeMap;
```

Comment:
You can avoid using a TMap here and just index directly into an array 

Worker_NetworkConnectionType ConnectionTypeMap[EWorkerType::Count];